relationships/synthesis/EXT-SERVICE-to-INFRA-AWSELASTICSEARCHCLUSTER.yml

```diff
@@ -1,13 +1,21 @@
 relationships:
-  - name: extServiceCallsElasticsearch
+  - name: extServiceCallsElasticsearch1_20
     version: "1"
     origins:
       - Distributed Tracing
+      - OpenTelemetry
     conditions:
       - attribute: eventType
         anyOf: [ "Span" ]
       - attribute: db.system
         anyOf: [ "elasticsearch" ]
+      - attribute: newrelic.source 
+        anyOf: [ "api.traces.otlp" ] 
+      - attribute: entity.type 
+        anyOf: [ "SERVICE" ] 
+      - attribute: net.peer.name  
+        present: true
+        regex: ^search-[a-zA-Z0-9-]+-[a-zA-Z0-9]+\.((aos\.[a-zA-Z0-9-]+\.on\.aws)|(([a-zA-Z0-9-]+)\.(es|aos)\.amazonaws\.com))$
```

Comment:
Is both `present` and `regex` necessary? I know we have done this on some other rules, but can we simplify and remove `present: true`?